A1_0_Compiling_Bitcoin_from_Source.md

```diff
@@ -0,0 +1,109 @@
+# Appendix I - Compiling Bitcoin from Source
+
+In this tutorial we will learn how to compile Bitcoin Core from source on a Debian based system. It is not necessary to compile bitcoin from source but, it has many benefits:
+
+1. You are always update to the latest release. Caveat: Being always updated is not necessary for Bitcoin Core since the software is always backwards compatible, meaning an old version of Bitcoin Core will still be able to participate in the Bitcoin network, though you may not have the latest features. You should always check the features of a new release before updating.
+2. You don't need to depend on pre-compiled Bitcoin Core binaries. This requires less trust. Even though the maintainers of Bitcoin Core do a great job of maintaining the integrity of the code, a pre-compiled binary is a few steps removed from source code. When you compile from the source code, the code is inspected before compilation.
+3. You can customize the build, like disabling the wallet or the GUI, etc.
+
+## Environment
+
+This tutorial uses Debian 10.4.kv0 OS on amd64 architecture(64 bit computers), but you can use this tutorial on any Debian based system e.g. Ubuntu, Mint, etc. On other linux systems, you can adapt the following steps with the package manager for that system.
+
+## Legend
+
+- Commands starting with `$` are executed as the user.
+- Commands starting with `#` are executed as root.
+- Text starting with `#` on the same line as the command are comments.
+
+Type the commands without `$` or `#`
+
+## Prerequisites
+
+- Basic or no familiarity with the command line and enthusiasm. The terminal is your most powerful ally, not something to be feared. You can simply copy and paste the following commands to compile bitcoin.
+- Admin previlages. If your user is not in the sudoers list then do the following otherwise skip this.
+
+```
+$ su -  # enter root. It will ask you for the root password.
+# apt-get install sudo  # check if sudo is installed.
+# usermod -aG <username> sudo  # add user to the sudoers list
+# reboot  # reboot computer
+```
```

Comment:
I don't think you have to reboot, just log the user out and in again. You should double-check that.

Reply:
I thought so too, but i double checked again and i had to reboot.

Reply:
Thanks for checking!